src/test/java/com/klarna/hiverunner/ReservedKeywordTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2013 Klarna AB
```

Comment:
No, they haven't changed, you just need to update the year. We use the Maven plugin that @teabot mentioned on our project to automate it, so you can define it in one place and then have it applied to all files. If you're interested in that I'd be happy to raise a PR with it.